sailor/assetcentral/system.py

```diff
@@ -116,17 +117,33 @@ def _component_tree(self):
         del self._components['key']
         return self._components
 
+    @cached_property
+    def components(self):
+        """Pieces of equipment that are children of the system.
+
+        Only top level, lower levels are ignored
+        """
+        warnings.warn("deprecated: attribute will be removed as soon as multilevel system hierarchies are supported",
+                      FutureWarning)
```

Comment:
I know we talked about it already and the decision was to add this warning.

But ultimately this is a confusing warning because `self.components` is also called by `get_indicator_data` and this warning will pop up all the time.

A FutureWarning should only be issued for deprecated features when our own code does not use it anymore. 

So I think we should remove the warning now, and bring it back when we have the replacement feature  available (delivering the components to the user as ResultSet).
@atrejut

Reply:
yes, agree